aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -350,13 +362,33 @@ module std::features {
         if (!exists<Features>(@std)) {
             move_to<Features>(framework, Features { features: vector[] })
         };
-        let features = &mut borrow_global_mut<Features>(@std).features;
-        vector::for_each_ref(&enable, |feature| {
-            set(features, *feature, true);
-        });
-        vector::for_each_ref(&disable, |feature| {
-            set(features, *feature, false);
-        });
+        apply_diff(borrow_global_mut<Features>(@std), enable, disable);
+    }
+
+    /// Function to enable and disable features. Can only be called by a signer of @std.
+    public fun change_feature_flags_for_next_epoch(framework: &signer, enable: vector<u64>, disable: vector<u64>)
```

Comment:
where is it used?

Reply:
it's supposed to be used by future governance proposals directly, right? @movekevin